docs/src/content/docs/es/reference/frontmatter.md

```diff
@@ -259,3 +259,29 @@ sidebar:
   hidden: true
 ---
 ```
+
+#### `badge`
+
+**tipo:** <code>string | <a href="/reference/configuration/#badgeconfig">BadgeConfig</a></code>
```

Comment:
Ooh, also noticed this link. Will get this fixed real quick and then LGTM. Thanks again @dreyfus92.

```suggestion
**tipo:** <code>string | <a href="/es/reference/configuration/#badgeconfig">BadgeConfig</a></code>
```